src/pocketmine/level/Level.php

```diff
@@ -932,7 +932,7 @@ public function clearCache(bool $force = false){
 			$this->chunkCache = [];
 			$this->blockCache = [];
 		}else{
-			if(count($this->blockCache) > 2048){
+			if(array_sum(array_map(function ($blockList) { return count($blockList); }, $this->blockCache)) > 2048){
```

Comment:
Is a lambda necessary here? `"count"` as a callable ought to suffice.

Reply:
ok, done

Reply:
now actually done. :p